clang/test/CodeGen/aie/aie2/aie2-upd-ext-intrinsic.cpp

```diff
@@ -1835,3 +1836,139 @@ v8bfloat16 test_extract_v8bfloat16_512(v32bfloat16 a, int idx) {
 v8bfloat16 test_extract_v8bfloat16_256(v16bfloat16 a, int idx) {
    return extract_v8bfloat16(a,idx);
 }
+
+// CHECK-LABEL: @_Z16test_set_v8floatiDv4_f(
```

Comment:
Nit: Would be nice to only push the diff for those new tests. THe lines changed above are just artifacts of the test updater script.

Reply:
yeah these changes are really annoying, and happen with almost every major LLVM version bump..
I'll revert these changes to reduce the noise